setup.py

```diff
@@ -1,16 +1,25 @@
 from setuptools import setup, find_packages
 
+requirements = [
+      "importlib_metadata",
+      "geopandas",
+      "pyyaml",
+      "loguru",
+      "requests",
+      "rtree",
+      "pyproj<3",
+]
+
+dev_packages = [
+      "pytest",
+]
+
 setup(name='openspoor',
       version='0.1.4',
```

Comment:
Raise version number? Or do we want to do this only for more major updates? Maybe we can discuss this next week.